agent/src/main/java/com/cloud/agent/dao/impl/PropertiesStorage.java

```diff
@@ -92,11 +92,11 @@ public synchronized boolean configure(String name, Map<String, Object> params) {
             file = new File(path);
             try {
                 if (!file.createNewFile()) {
-                    s_logger.error("Unable to create _file: " + file.getAbsolutePath());
+                    s_logger.error("Unable to create _file: ".concat(file.getAbsolutePath()));
```

Comment:
the preferred way would be to use String.format, the concatenation using `+` isn't any different from concat.

Reply:
thank you for your suggestion , I will change it .